presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -912,6 +912,40 @@ procedure on the catalog's ``system`` schema::
     ``unregister_table`` only when using the Hive catalog. This is similar to
     the behavior listed above for the ``DROP TABLE`` command.
 
+Expire snapshots
+^^^^^^^^^^^^^^^^
+
+Each dml action in Iceberg produces a new snapshot while keeping the old data and metadata around for snapshot isolation and time travel. The expire_snapshots procedure can be used to remove older snapshots and their files which are no longer needed.
+
+This procedure will remove old snapshots and the corresponding snapshot files, and it will never remove files which are still required by a non-expired snapshot.
+
+The following arguments are available:
+
+===================== ========== =============== =======================================================================
+Argument Name         required   type            Description
+===================== ========== =============== =======================================================================
+``schema``            ✔️         string          Schema of the table to update
+
+``table_name``        ✔️         string          Name of the table to update
+
+``older_than``                   timestamp       Timestamp before which snapshots will be removed (Default: 5 days ago)
+
+``retain_last``                  int             Number of ancestor snapshots to preserve regardless of older_than
+                                                 (defaults to 1)
+
+``snapshot_ids``                 array of long   Array of snapshot IDs to expire
```

Comment:
Is a word missing after "array of long"?

Reply:
No, what I want to express is that the type is `array of long`. Is it OK with this expression in your opinion?

Reply:
That's fine! I was unfamiliar with the phrase and I should have looked it up first. This is good as is. Thanks!